### PR TITLE
Cleaning up imports and removing doublets from mouse data

### PR DESCRIPTION
**Pull request by nbedanova**
There are quite a few imports I do not use in the imports file that I deleted. Unsure if we want to keep them. Also, there are cells labeled as doublets in the mouse cytokine data so removing that when importing. Also removing regularization parameter variable in the pf2 function.

### CODE REVIEW

File: pf2rnaseq/imports.py [hunk 1]
```diff
@@ -1,15 +1,53 @@
...
 import numpy as np
 import pandas as pd
 import scanpy as sc
-from scipy.sparse import csr_matrix, spmatrix
+from scipy.sparse import csr_array, spmatrix
+from sklearn.preprocessing import scale
 from sklearn.utils.sparsefuncs import inplace_column_scale, mean_variance_axis
 
 
+def prepare_dataset_deviance(
```
Comment (aarmey) [inline pf2rnaseq/imports.py:12]: I *think* that `prepare_dataset()`, both with and without the deviance approach, should be correctly implemented in the parafac2 repository now. https://github.com/meyer-lab/parafac2/blob/main/parafac2/normalize.py

Can you import it from there? It'd be better to reuse this code.

**Review by aarmey — CHANGES_REQUESTED** [review]

File: pf2rnaseq/imports.py (hunk 1 above)
Comment (nbedanova) [reply pf2rnaseq/imports.py:12]: Getting a few errors when trying to use deviance transform, trying to parse through and see what the issue is. I think that pi_j[0, col] needs to be reshaped to a row vector as there is a dimension issue in the mu_ij_nn = n_i_col[row, 0] * pi_j[0, col] calculation.  However after doing that the term1 = data.data * np.log(data.data / mu_ij_nn) calulation is giving me errors due to the difference in size of data.data and mu_ij_nn. Also doesn't pi_j need to be a proportion so divided by counts?

Comment (aarmey) [reply pf2rnaseq/imports.py:12]: Can you update this code to use the parafac2 repo function? After that, I'll look into these errors.

Comment (nbedanova) [reply pf2rnaseq/imports.py]: Done
